dir.props

```diff
@@ -10,9 +10,11 @@
   </PropertyGroup>
 
   <PropertyGroup>
+    <TargetFrameworkVersion>v4.5.1</TargetFrameworkVersion>
```

Comment:
Bad merge. This got updated in master and moved in xplat and somehow I came out of it with the old value :disappointed: